redisinsight/ui/src/pages/vector-search/create-index/hooks/useRedisInstanceCompatibility.ts

```diff
@@ -15,10 +15,13 @@ const REDISEARCH_MODULE_SET = new Set(REDISEARCH_MODULES)
 
 const useRedisInstanceCompatibility =
   (): UseRedisInstanceCompatibilityReturn => {
+    const {
+      version,
+    } = useSelector(connectedInstanceInfoSelector)
```

Comment:
Ouch, this means we have different behavior for what we store in Redux, based on the environment 😔